arches/app/templates/views/components/datatypes/resource-instance.htm

```diff
@@ -35,13 +35,19 @@
                             <i class="fa fa-trash"></i>
                         </button>
                     </div>
-                    <div class="rr-table-column" tabindex="0" role="button" style="flex-grow: 1; padding-left: 10px;"
-                        data-bind="
+                    <div class='rr-table-column icon-column'>
+                        <button 
+                            data-bind="
                             click: function() { if($parent.isEditable !== false){self.toggleSelectedResource($data)}; },
                             clickBubble: false,
                             onEnterkeyClick, onSpaceClick," 
-                        style="cursor: pointer;">
+                            style="cursor: pointer;">
+                            <i class="fa fa-gear"></i>
+                        </button>
```

Comment:
When the node is not editable, we should style the icon as disabled, just like delete:

<img width="590" alt="disabled-style" src="https://github.com/archesproject/arches/assets/38668450/100458cd-6391-4f98-830d-1eadebf0004a">

Reply:
fix in latest